tests/integration_tests/performance/test_block_ab.py

```diff
@@ -142,7 +142,7 @@ def process_fio_logs(vm, fio_mode, logs_dir, metrics):
 @pytest.mark.parametrize("fio_block_size", [4096], ids=["bs4096"])
 def test_block_performance(
     microvm_factory,
-    guest_kernel,
+    guest_kernel_acpi,
```

Comment:
hmm we will have to clean this up later, but it beats bringing the condition here.

Reply:
Mh, true, but hopefully the commit will still revert cleanly once we drop the no-acpi kernels